microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -111,15 +111,16 @@ In the above example, a music player application defines a mini player widget. A
 | `name` | The title of the widget, presented to users. | Yes |
 | `short_name` | An alternative short version of the name. | No |
 | `description` | A description of what the widget does. | Yes |
-| `icons` | An array of icons to be used for the widget. If missing, the `icons` manifest member is used instead. | No |
-| `screenshots` | An array of screenshots that show what the widget looks like. Analogous to the [`screenshot` manifest member](https://developer.mozilla.org/docs/Web/Manifest/screenshots). Note that the `platform` field of a screenshot item currently supports the `Windows` and `any` values. | Yes |
+| `icons` | An array of icons to be used for the widget. If missing, the `icons` manifest member is used instead. Icons larger than 1024x1024 are ignored. | No |
+| `screenshots` | An array of screenshots that show what the widget looks like. Analogous to the [`screenshot` manifest member](https://developer.mozilla.org/docs/Web/Manifest/screenshots). The `platform` field of a screenshot item supports the `Windows` and `any` values. Images larger than 1024x1024 pixels are ignored. For screenshot requirements specific to the Windows 11 Widgets Board, see [Screenshot image requirements](/windows/apps/design/widgets/widgets-picker-integration#screenshot-image-requirements) in _Integrate with the widget picker_. | Yes |
 | `tag` | A string used to reference the widget in the PWA service worker. | Yes |
 | `template` | The template to use to display the widget in the operating system widgets dashboard. Note: this property is currently only informational and not used. See `ms_ac_template` below. | No |
 | `ms_ac_template` | The URL of the custom Adaptive Cards template to use to display the widget in the operating system widgets dashboard. See [Define a widget template](#define-a-widget-template) below. | Yes |
 | `data` | The URL where the data to fill the template with can be found. If present, this URL is required to return valid JSON. | No |
 | `type` | The MIME type for the widget data. | No |
 | `auth` | A boolean indicating if the widget requires authentication. | No |
 | `update` | The frequency, in seconds, at which the widget will be updated. Code in your service worker must perform the updating; the widget is not updated automatically. See [Access widget instances at runtime](#access-widget-instances-at-runtime). | No |
+| `multiple` | A boolean indicating whether to allow multiple instances of the widget. Defaults to true. | No |
```

Comment:
```suggestion
| `multiple` | A boolean indicating whether to allow multiple instances of the widget. Defaults to `true`. | No |
```
Backticks to mark "true" as code rather than regular words.  d/k if you saw previous comment about this and want to keep it plain text; re-entering in case missed.

Reply:
Yes sorry, I missed the backticks on the original comment and made the change manually since I wasn't logged in with the right account at the time. Thanks for catching that!